Controller/Adminhtml/Entity/MassDelete.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Smile\CustomEntity\Controller\Adminhtml\Entity;
+
+use Magento\Backend\App\Action;
+use Magento\Backend\App\Action\Context;
+use Magento\Framework\App\Action\HttpPostActionInterface;
+use Magento\Framework\Controller\ResultFactory;
+use Magento\Ui\Component\MassAction\Filter;
+use Smile\CustomEntity\Model\ResourceModel\CustomEntity\CollectionFactory;
+
+/**
+ * Mass delete custom entity controller
+ */
+class MassDelete extends Action implements HttpPostActionInterface
+{
+    protected $filter;
```

Comment:
Can you add annotation here to type these properties ?

Reply:
Updated